src/integrationTest/java/uk/gov/hmcts/reform/demo/controllers/GetWelcomeTest.java

```diff
@@ -1,26 +1,25 @@
 package uk.gov.hmcts.reform.demo.controllers;
 
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
-import org.springframework.test.context.junit4.SpringRunner;
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.MvcResult;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-@RunWith(SpringRunner.class)
 @WebMvcTest
 public class GetWelcomeTest {
 
     @Autowired
     private transient MockMvc mockMvc;
 
+    @DisplayName("Should welcome upon root request with 200 response code")
     @Test
-    public void should_welcome_upon_root_request_with_200_response_code() throws Exception {
+    public void welcomeRootEndpoint() throws Exception {
```

Comment:
imho test method name was perfectly readable already.

Reply:
Display name is much nicer imo